.github/workflows/send_email.yml

```diff
@@ -0,0 +1,38 @@
+name: Release Available Notice
+on:
+  # Automated runs
+  release:
+    types: [published]
+
+  # Manual runs
+  workflow_dispatch:
+
+jobs:
+  email:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Send mail
+        uses: dawidd6/action-send-mail@v3
+        with:
+          server_address: ${{ secrets.MAIL_SERVER}}
```

Comment:
So, not sure if either of you have gotten permissions for adding secrets to this repo yet? I cannot.

If not, this workflow will not be able to execute. So, can either not merge this in till we are able to configure secrets. Or, can change the trigger to not have the automatic runs until we can configure secrets.